### PR TITLE
Update README.md for now

### PR DESCRIPTION
**Pull request by vivekvbajpai**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/vivekvbajpai/Academic_Projects/pull/2" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ CS579_assignments--
 
 CS546_assignments--
 	Project and assignments done for "Parallel and Distributed Processing" course.
-	Technologies- C, MPI, OpenMP, Pthread, CUDA
```
Comment (vivekvbajpai) [inline README.md:10]: !fix

Comment (vivekvbajpai) [inline README.md:10]: !resolve

Comment (vivekvbajpai) [inline README.md:10]: !fixed

File: README.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ CS579_assignments--
...
\ No newline at end of file
+	Technologies- C, MPI, OpenMP, Pthread, CUDA.
```
Comment (vivekvbajpai) [inline README.md:10]: wrrrrrrong

**Pull request merged**